src/main/java/frc/robot/mechanisms/pivot/Pivot.java

```diff
@@ -39,7 +41,7 @@ public PivotConfig() {
             configSupplyCurrentLimit(currentLimit, threshold, true);
             configNeutralBrakeMode(true);
             configCounterClockwise_Positive(); // TODO: configure
-            configReverseSoftLimit(minRotation, true);
+            configReverseSoftLimit(minRotation, false);
```

Comment:
both soft limits have to be enabled, without them the pivot can move into the phsyical hardstops and will break things (esp cuz operator and pilot have manual pivot control, we can't rely solely on setpoints being within our softlimits) 

it's very rare that we won't have both soft limits enabled on a mechanism that's not just rollers